runtime/doc/lua.txt

```diff
@@ -1240,19 +1240,36 @@ inspect({object}, {options})                                   *vim.inspect()*
                     https://github.com/kikito/inspect.lua
                     https://github.com/mpeterv/vinspect
 
-notify({msg}, {log_level}, {opts})                              *vim.notify()*
-                Notification provider
+notify({msg}, {level}, {opts})                                  *vim.notify()*
+                Display a notification to the user.
 
-                Without a runtime, writes to :Messages
+                This function can be overridden by plugins to display
+                notifications using a custom provider (such as the system
+                notification provider). By default, writes to :messages.
 
                 Parameters: ~
-                    {msg}        string Content of the notification to show to
-                                 the user
-                    {log_level}  number|nil enum from vim.log.levels
-                    {opts}       table|nil additional options (timeout, etc)
+                    {msg}    string Content of the notification to show to the
+                             user.
+                    {level}  number|nil One of the values from
+                             |vim.log.levels|.
```

Comment:
Thanks for making this a ref. I've been annoyed a few times by not being able to easily nav to this.